numba_cuda/numba/cuda/random.py

```diff
@@ -3,18 +3,21 @@
 
 import math
 
-from numba import (
-    cuda,
+from numba import cuda
+from numba.cuda import (
     float32,
     float64,
     uint32,
     int64,
     uint64,
     from_dtype,
```

Comment:
This should come from `numba.cuda.np.numpy_support` so that it doesn't need to be brought into the top level `numba.cuda` package.